pandas/core/indexing.py

```diff
@@ -228,7 +228,9 @@ def _validate_key(self, key, axis: int):
         raise AbstractMethodError(self)
 
     def _has_valid_tuple(self, key: Tuple):
```

Comment:
might as well annotate while in the neighborhood

Reply:
never mind, for a second github was only showing me this one piece of the diff

Reply:
> might as well annotate while in the neighborhood

I don't see it returning anything, annotate it as ```-> None```?

Reply:
pls ignore, i was looking at the wrong diff

Reply:
I know, but I saw other functions that need to annotated.